src/material-examples/text-field-autofill-monitor/text-field-autofill-monitor-example.ts

```diff
@@ -8,8 +8,8 @@ import {Component, ElementRef, OnDestroy, OnInit, ViewChild} from '@angular/core
   styleUrls: ['./text-field-autofill-monitor-example.css'],
 })
 export class TextFieldAutofillMonitorExample implements OnDestroy, OnInit {
-  @ViewChild('first', {read: ElementRef}) firstName: ElementRef<HTMLElement>;
-  @ViewChild('last', {read: ElementRef}) lastName: ElementRef<HTMLElement>;
+  @ViewChild('first', {read: ElementRef, static: true}) firstName: ElementRef<HTMLElement>;
+  @ViewChild('last', {read: ElementRef, static: true}) lastName: ElementRef<HTMLElement>;
```

Comment:
Can we move the logic to `ngAfterViewInit` instead of making these queries static? I think we want to try to use static queries only when they're necessary.

Reply:
Makes sense, updated. Thnx @kara